<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Admin" failures="1" tests="9" name="TestNg.AttributeBasics" time="0.054" errors="0" timestamp="2024-08-17T13:34:07 IST" skipped="2">
  <testcase classname="TestNg.AttributeBasics" name="test7" time="0.001"/>
  <testcase classname="TestNg.AttributeBasics" name="test3" time="0.004"/>
  <testcase classname="TestNg.AttributeBasics" name="test4" time="0.017"/>
  <testcase classname="TestNg.AttributeBasics" name="test8" time="0.011"/>
  <testcase classname="TestNg.AttributeBasics" name="test5" time="0.001"/>
  <testcase classname="TestNg.AttributeBasics" name="test7" time="0.001"/>
  <testcase classname="TestNg.AttributeBasics" name="test2" time="0.019">
    <failure message="Values not matched expected [20] but found [10]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Values not matched expected [20] but found [10]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:1419)
at org.testng.Assert.assertEquals(Assert.java:1383)
at TestNg.AttributeBasics.test2(AttributeBasics.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
    </failure>
  </testcase> <!-- test2 -->
  <testcase classname="TestNg.AttributeBasics" name="test1">
    <skipped/>
  </testcase> <!-- test1 -->
  <testcase classname="TestNg.AttributeBasics" name="test6">
    <skipped/>
  </testcase> <!-- test6 -->
</testsuite> <!-- TestNg.AttributeBasics -->
